### PR TITLE
[FIX] Image Embedding: does not crash on empty data

### LINKED ISSUE #45: Image Embedding: crashes when nothing in dropdown
##### Orange version
master

##### Image Analytics version
0.1.7

##### Expected behavior
Does not crash.

##### Actual behavior
screenshot_20170609_152511

##### Steps to reproduce the behavior
See a screenshot above.

### PR DESCRIPTION
**Pull request by jerneju**
Fixes #45 .

### CODE REVIEW

**Comment by inejc** [comment]
The zero length check should also be added to the `set_data` method.

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3-imageanalytics/issues/290587305/

File: orangecontrib/imageanalytics/widgets/owimageembedding.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def _init_server_connection(self):
         )
 
     def set_data(self, data):
-        if data is None:
+        if data is None or not len(data):
```
Comment (inejc) [inline orangecontrib/imageanalytics/widgets/owimageembedding.py]: My comment was regarding the same check as in the `commit` method (zero length of `self._image_attributes`). Below the changed if statement here is a similar check where the _"..., but without image attributes."_ text is displayed.

Comment (jerneju) [inline orangecontrib/imageanalytics/widgets/owimageembedding.py]: Fixed.

**Pull request merged**